src/main/java/com/bookshop/BookShopApplication.java

```diff
@@ -1,12 +1,39 @@
 package com.bookshop;
 
+import com.bookshop.model.Book;
+import com.bookshop.service.BookService;
+import java.math.BigDecimal;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
 
 @SpringBootApplication
 public class BookShopApplication {
+    private final BookService bookService;
+
+    @Autowired
+    public BookShopApplication(BookService bookService) {
```

Comment:
When using Lombok, you can simply annotate a class with `@RequiredArgsConstructor` and not create constructor manually.